infra/scripts/bash-helpers.sh

```diff
@@ -48,7 +48,7 @@ vic-create () {
     base=$(pwd)
     (
         cd "$(vic-path)"/bin || return
-        "$(vic-path)"/bin/vic-machine-"$OS" create --target="$TARGET_URL" "${OPS_CREDS[@]}" --image-store="$IMAGE_STORE" --compute-resource="$COMPUTE" "${TLS[@]}" ${TLS_OPTS} --name="${VIC_NAME:-${USER}test}" "${MAPPED_NETWORKS[@]}" "${VOLUME_STORES[@]}" "${NETWORKS[@]}" ${IPADDR} ${TIMEOUT} --thumbprint="$THUMBPRINT" "$@"
+        "$(vic-path)"/bin/${VIC_VERSION}/vic-machine-"$OS" create --target="$TARGET_URL" "${OPS_CREDS[@]}" --image-store="$IMAGE_STORE" --compute-resource="$COMPUTE" "${TLS[@]}" ${TLS_OPTS} --name="${VIC_NAME:-${USER}test}" "${MAPPED_NETWORKS[@]}" "${VOLUME_STORES[@]}" "${NETWORKS[@]}" ${IPADDR} ${TIMEOUT} --thumbprint="$THUMBPRINT" --ai=${VIC_VERSION}/appliance.iso --bi=${VIC_VERSION}/bootstrap.iso "$@"
```

Comment:
> `"$(vic-path)"/bin/${VIC_VERSION}/vic-machine-"$OS"`

Perhaps it would be safer to just quote this whole string, to avoid potential issues with spaces appearing in `VIC_VERSION`:

`"$(vic-path)/bin/${VIC_VERSION}/vic-machine-$OS"`

(Also applies to the analogous modifications in this PR.)

> --ai=${VIC_VERSION}/appliance.iso --bi=${VIC_VERSION}/bootstrap.iso

For safety and consistency with handling of other arguments, I'd quote these as well:

`--ai="${VIC_VERSION}/appliance.iso" --bi="${VIC_VERSION}/bootstrap.iso"`